react/features/invite/components/AddPeopleDialog.web.js

```diff
@@ -84,8 +91,20 @@ class AddPeopleDialog extends Component {
 
         this._multiselect = null;
         this._resourceClient = {
-            makeQuery: text => searchPeople(
-                this.props._peopleSearchUrl, this.props._jwt, text),
+            makeQuery: text => {
+                const {
+                    _jwt,
+                    _peopleSearchQueryTypes,
+                    _peopleSearchUrl
+                } = this.props;
+
+                return searchPeople(
+                    _peopleSearchUrl,
+                    _jwt,
+                    text,
+                    _peopleSearchQueryTypes
```

Comment:
I would put the peopleSearchQueryTypes parameter after the peopleSearchUrl or at least before the actual text because it logically is a meta data we need to pass before the actual query text.

Reply:
Will fix it!